src/pages-and-resources/discussions/data/thunks.js

```diff
@@ -94,3 +98,9 @@ export function updatedDiscussionTopics(payload) {
     dispatch(updateModels({ modelType: 'discussionTopics', models: payload }));
   };
 }
+
+export function updateAppConfigs(payload) {
+  return async (dispatch) => {
```

Comment:
why this is async?

Reply:
As i figure out there is no side effect calls involve so we need to move this dispatch call in component.